### PR TITLE
Added a lint rule to check if Neutral is set up correctly

### LINKED ISSUE #8269: Write a lint rule to verify system player stances
All maps should have a Neutral actor that defines OwnsWorld: true and has no allies or enemies.  Maps that define a Creeps player should have them enemy to everyone.  This will prevent any new bugs like #8263.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/17888946-write-a-lint-rule-to-verify-system-player-stances?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Closes #8269

### CODE REVIEW

**Comment by chrisforbes** [comment]
:+1: This looks OK to me.

As a followup PR, do you want to insist on there being exactly one player with OwnsWorld=true?

**Comment by Mailaender** [comment]
I haven't thought about that being a problem. Anyway, let's do it in a followup. This rule already detected enough errors for this pull request to review.

**Comment by pchote** [comment]
Some of these missions depend on the current behavior to produce their gameplay (e.g. tesla coils attacking the civilians in allies01).  The mission scripting will need to be changed / new playerreferences added in order to not break these cases.

**Comment by Mailaender** [comment]
The civilians in allies-01 or on England's side.

File: OpenRA.Mods.Common/Lint/CheckPlayers.cs [hunk 1]
```diff
@@ -22,22 +22,26 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
...
+			}
+
 			var worldActor = map.Rules.Actors["world"];
 
 			var races = worldActor.Traits.WithInterface<CountryInfo>().Select(c => c.Race);
-			foreach (var player in players)
-				if (!string.IsNullOrWhiteSpace(player.Value.Race) && player.Value.Race != "Random" && !races.Contains(player.Value.Race))
-					emitError("Invalid race {0} chosen for player {1}.".F(player.Value.Race, player.Value.Name));
+			foreach (var player in players.Values)
+				if (!string.IsNullOrWhiteSpace(player.Race) && player.Race != "Random" && !races.Contains(player.Race))
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Lint/CheckPlayers.cs]: The hard-coded check for the random race isn't needed anymore. "Random" is a legit race in its own right now.

**Comment by obrakmann** [comment]
Tested all affected missions, noticed no regressions. :+1: after the random check has been removed.

**Comment by Mailaender** [comment]
Yep, looks like this was forgotten in https://github.com/OpenRA/OpenRA/pull/7476. Updated.

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/d64a571c2c072302fa0040d45329e0f75ade160e...be092fc34ba4db26772dc28aa1a561b55eafbf0f)

**Pull request merged**